### ISSUE TITLE
Encoding "no physical parameters" in Trajectory files

### ISSUE THREAD

**Issue by MarkIggy** [euroargodev/ArgoNetCDF#22]
How should we encode "no physical parameters" (no <PARAM> variables) in the Trajectory files?  There will be cases where there is no "physical parameter" data to record in a Trajectory file but there is still other valuable information.

We have two options:
1) N_PARAM = 1; TRAJECTORY_PARAMETERS=  [" "]; no <PARAM> variables
2) No N_PARAM, TRAJECTORY_PARAMETERS, or <PARAM> variables in the file

Which would be more instructive to the user?

(They may not be mutually exclusive.  Could/should we allow both?)

**Comment by jegilson** [comment]
PRES is a fundamental dimension of the trajectory file, 2nd only to TIME.  I would suggest PRES must be included (N_PARAM=1 minimum), and TRAJECTORY_PARAMETERS must include at least PRES, matching the N_PARAM definition.  The PRES variables can be empty.  Although it is hard for me to think that this is possible, even back in the earliest floats there was a few PRES data.  In any case, the lack of any PRES information supplies the user information on the validity of whatever remains from a sparse traj netCDF as is described here.

**Comment by ClaudiaAOML** [comment]
we had a handful of protoptype floats long ago that did not report any pressure data during drift. Because that makes it hard to use the trajectory for deriving currents at a known depth we did not make trajectory files for these floats.

In addition to that, they aslo had only one position per cycle, which limits the usefullness of trajectory files even more.

**Comment by mscanderbeg** [comment]
I agree with John's comment and that we should have PRES, even if it is empty.  This seems like a modified Option 1:
N_PARAM = 1; TRAJECTORY_PARAMETERS= ["PRES"]; PRES may be fillvalue
